wandb/sdk/launch/agent/agent.py

```diff
@@ -342,8 +348,15 @@ def thread_run_job(
     ) -> None:
         try:
             self._thread_run_job(launch_spec, job, default_config, api)
```

Comment:
so in case it fails you don't error out?

Reply:
Right - if the run fails to start, we still want to ack the run queue item so that it's not picked up by the agent again